jujugui/static/gui/src/app/components/user-profile/model-list/model-list.js

```diff
@@ -45,11 +47,10 @@ YUI.add('user-profile-model-list', function() {
     },
 
     getInitialState: function() {
-      this.xhrs = [];
```

Comment:
Pulling xhr stuff out was sort of a drive-by cleanup. The controller API calls happen via a websocket, which doesn't return any value, so we're essentially storing an array of nulls right now. I've talked with @hatched and @frankban about how to gracefully cancel any in-flight websockets in response to a componentWillUnmount(), but until that is in place, there's no point having this code for the time being.